components/ImaSdkTask.brs

```diff
@@ -244,11 +192,6 @@ sub setupVideoPlayer()
 
     m.player.loadUrl = function(urlData)
         ? "TRUE[X] >>> ImaSdkTask::loadUrl(urlData=";urlData;")"
-        ' if m.streamManager <> invalid and m.top.streamData.type <> "live" then
-        '     m.top.bookmark = m.streamManager.GetStreamTime(m.top.bookmark * 1000)
-        ' else
-        '     m.top.bookmark = 0
-        ' end if
```

Comment:
you have a commented out code block here, remove?

Reply:
🤣